quipucords/scanner/openshift/inspect.py

```diff
@@ -80,6 +81,10 @@ def _extra_cluster_facts(
                 )
             except OCPError as err:
                 cluster.errors[fact_name] = err
+
+        # Let's add the cluster metrics facts
+        extra_facts.update(metrics.retrieve_cluster_metrics(ocp_client))
```

Comment:
OCP scanner has a tradition of collecting errors as part of the raw facts. If you follow the suggestion to treat the exception via catch_k8s_exception decorator (https://github.com/quipucords/quipucords/pull/2438#discussion_r1288942225) you could do the same try/except we do above

Reply:
Gotcha, thank you.  I will give it a try.